modules/ROOT/pages/streaming-about.adoc

```diff
@@ -3,64 +3,101 @@ ifndef::env-site,env-github[]
-
-You can configure how Mule handles the repeatable stream by using streaming strategies.
+As a component consumes the stream, Mule saves its content into a temporary
+buffer. The runtime then feeds the component from the temporary buffer, ensuring
+that each component receives the full stream, regardless of how much of the
+stream was already consumed by any prior component. This happens automatically
+and requires no special configuration by you, which prevents the need to find
+workarounds to save the stream elsewhere so you can access it again. This
+configuration automatically fixes the first two Mule 3 examples outlined above.
+
+All repeatable streams support parallel access, which means that you don't need
+to worry about whether two components are trying to read the same stream when
+each component is running on a different thread. Mule automatically ensures
+that when component `A` reads the stream it doesn’t generate any side effects
+in component `B`. This behavior enables you to perform tasks like the one
+described in the <<example_3, third example>>.
 
 == Streaming Strategies
 
-=== File Stored Repeatable Stream
+You can configure how Mule handles streams with streaming strategies.
+
+=== File-Stored Repeatable Stream
 
-This is the default streaming strategy in Mule Runtime Enterprise Edition.
+File storage is the default streaming strategy in Mule 4.
 
 [NOTE]
 --
-This option is only available on Mule EE
+File storage is only available in Mule Enterprise Edition (Mule EE).
 --
 
-It initially uses an in-memory buffer size of 512 KB. If the stream is larger than that, it creates a temporary file on your disk to store the contents without overflowing your memory.
+This strategy initially uses an in-memory buffer size of 512 KB. For larger
+streams, the strategy creates a temporary file on your disk to store
+the contents, without overflowing your memory.
 
-When you know you need to deal with large or small files, you can change the buffer size to optimize performance. +
-Configuring a bigger buffer size increases performance by avoiding the number of times the runtime needs to write the buffer to your disk, but it also limits the number of concurrent requests your application can process. +
-The same way, configuring a smaller buffer size saves you memory load. +
-You can even set the buffer's unit of measurement, so you don't have to go through unit conversions.
+If you need to handle large or small files, you can change the buffer size
+(`inMemorySize`) to optimize performance:
 
-For example, if you know that you are going to read a file that's always around 1 MB size, you can configure a 1 MB buffer:
+* Configuring a larger buffer size increases performance by avoiding the number
+of times the runtime needs to write the buffer to your disk, but it also limits
```

Comment:
to your disk > to disk. 

I'd remove **your** since this could be also configured for a CH worker (I think).

Reply:
revising, thx